config/routes.rb

```diff
@@ -55,5 +55,9 @@
   #   end
 
   root 'home#index'
-  resources :cards
+  resources :cards do
+    resources :trainer, only: [:review] do
+      patch 'review', on: :collection
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.